src/services/importer.js

```diff
@@ -24,7 +25,13 @@ export const convert = (schema, markdown) => {
       const [segmentKey] = segment.match(/^.*$/m)
       const segmentValue = segment.replace(segmentKey, '')
       if (segmentKey.trim().length) {
```

Comment:
I guess that this could just be `if (segmentKey.trim())`, but it's already there and works so forget it